ospfd/ospf_vty.c

```diff
@@ -4717,34 +4717,44 @@ static int show_ip_ospf_neighbor_int_common(struct vty *vty, struct ospf *ospf,
 
 DEFUN (show_ip_ospf_neighbor_int,
        show_ip_ospf_neighbor_int_cmd,
-       "show ip ospf neighbor IFNAME [json]",
+       "show ip ospf [vrf <NAME>] neighbor IFNAME [json]",
        SHOW_STR
        IP_STR
        "OSPF information\n"
+       VRF_CMD_HELP_STR
        "Neighbor list\n"
        "Interface name\n"
        JSON_STR)
 {
 	struct ospf *ospf;
-	int idx_ifname = 4;
+	int idx_ifname = 4, idx_vrf;
 	bool uj = use_json(argc, argv);
```

Comment:
Please put the declaration of `idx_vrf` varaible on a separate line for better lisibility